src/extractors/custom/www.infoq.com/index.js

```diff
@@ -0,0 +1,31 @@
+export const WwwInfoqComExtractor = {
+  domain: 'www.infoq.com',
+
+  title: {
+    selectors: ['h1.heading'],
+  },
+
+  author: {
+    selectors: ['div.widget.article__authors'],
+  },
+
+  date_published: null,
```

Comment:
Could you please test querying the date using something like the following code:
```javascript
  date_published: {
    selectors: ['.article__readTime.date'],
    format: 'YYYY年MM月DD日',
  },
```

Reply:
Thank you! I fixed.